docs/reference/data-streams/set-up-a-data-stream.asciidoc

```diff
@@ -106,6 +106,15 @@ template for a data stream must specify:
 
 * That the template is used exclusively for data streams.
 
+* A priority for the template.
+
+IMPORTANT: {es} comes with built-in index templates for the `metrics-*-*` and
+`logs-*-*` index patterns.
+{ingest-guide}/ingest-management-overview.html[{agent}] uses these templates to
+create data streams. To avoid accidentally applying these templates, use a
```

Comment:
> Can a data stream template with an index pattern that overlaps the built-in templates also inadvertently override the built-in templates? For example, if I add a data stream template with the index pattern logs* and give it a higher priority than the built in template (presumably 100?), have I just messed things up for the Elastic Agent's logs data stream?

Yes, if someone put a template in place with a higher priority it would hijack the built-in templates. Fleet does this to override them with more specific templates when it installs its templates

Reply:
Added this sentence to address people using Elastic Agent:

```
If you use Elastic Agent, assign your index templates a priority lower than `100` to avoid an override of the built-in templates.
```